src/ExtranetBundle/objects/db/TDataExtranetUser.class.php

```diff
@@ -960,7 +974,7 @@ public function ValidateSessionData()
             if ($aUserData) {
                 $iCurrentTime = time();
                 $dataValid = (($iCurrentTime - $aUserData['login_timestamp']) < $oExtranetConfig->fieldSessionlife);
-                if ($dataValid && CHAMELEON_EXTRANET_USER_IS_PORTAL_DEPENDANT && !TGlobal::IsCMSMode()) {
+                if ($dataValid && CHAMELEON_EXTRANET_USER_IS_PORTAL_DEPENDANT && !TGlobal::IsCMSMode() && !$this->isRequestForResource()) {
```

Comment:
Checking for example for the "master request" doesn't help here?

Reply:
This IS the master request. We are not dealing with sub requests here. It is the actual request done by the browser when loading a resource referenced in the html